design/ceph/ceph-zone.md

```diff
@@ -0,0 +1,183 @@
+# Rook Ceph Zone
+
+## Prerequisites
+
+A Rook Ceph cluster. Ideally a ceph-realm and a ceph-zone-group resource would have been started up already.
+
+## Ceph Multisite Overview
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+For more information on Ceph multsite, please read the [ceph-realm](/design/ceph/ceph-realm.md) design document or [Ceph's documentation](http://docs.ceph.com/docs/master/radosgw/multisite/) on the feature.
+
+## Ceph Zone Walkthrough
+
+The resource descibed in this design document represents the zone in the Ceph Multisite data model.
+
+### Creating an Ceph Zone
+
+#### Config
+
+When the storage admin is ready to create a multisite zone for object storage, the admin will name the zone in the metadata section on the configuration file.
+
+In the config, the admin must configure the zone group the zone is in, and pools for the zone.
+
+The first zone created in a zone group is designated as the master zone.
```

Comment:
the master zone declaration is removed now, right?

Reply:
Yup